### ISSUE TITLE
[Bug]: Wrongly use batchnorm2d

### ISSUE THREAD

**Issue by heqin-zhu** [shiba24/3d-unet#10]
https://github.com/shiba24/3d-unet/blob/398dca9b1752a07a7f80c45bbca017ca0fedeed5/pytorch/model.py#L39
Should we use batchnorm3d instead of batchnorm2d in the referred line?

**Comment by shiba24** [comment]
yeah i guess so. could you create a PR?

**Comment by heqin-zhu** [comment]
Okay! I will do it later.

**Comment by shiba24** [comment]
fixed with #11
